### PR TITLE
Add German translations

### PR DESCRIPTION
**Pull request by MinThaMie**

### CODE REVIEW

File: translations/de.yaml [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+      title: 'Schüler'
+      description: 'Ich lerne mehr über unbewusste Vorurteile und ihre Auswirkungen.'
+      explanation: ''
+    globetrotter:
+      title: 'Weltbummler'
+      description: 'Ich tausche mich mit internationalen Kollegen aus und bin neugierig auf ihre Perspektiven.'
+      explanation: ''
+
+glossary:
+  subtitle: 'Hier überprüfen'
```
Comment (robinaly) [inline translations/de.yaml]: Better: "Bitte überprüfen"

File: translations/de.yaml [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+  title: 'DEI-Vokabular'
+  welcome: 'Sie sind sich nicht sicher, was einige Begriffe bedeuten? Kein Problem! In diesem Überblick finden Sie viele Begriffe, die Ihnen in Ihrer Organisation begegnen können.'
+  placeholder: 'Hier eingeben, um das Glossar zu filtern'
```
Comment (robinaly) [inline translations/de.yaml]:
```suggestion
  placeholder: 'Hier tippen, um das Glossar zu filtern'
```

File: translations/de.yaml [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+    inclusion:
+      title: 'Inklusion'
+      definition: 'Die Handlung, Umgebungen zu schaffen, in denen jedes Individuum oder jede Gruppe sich willkommen, respektiert, unterstützt und geschätzt fühlen kann, um vollständig teilzunehmen. Ein inklusives und einladendes Klima akzeptiert Unterschiede und drückt Respekt in Worten und Taten für alle Menschen aus.'
+    intersectionality:
+      title: 'Intersektionalität'
+      definition: 'Ein analytischer Rahmen, um zu verstehen, wie verschiedene Aspekte der sozialen und politischen Identität einer Person (z. B. Geschlecht, Rasse, Klasse, Sexualität, Fähigkeit, Aussehen usw.) zusammentreffen und einzigartige Kombinationen von Diskriminierung und Privileg erzeugen. Intersektionalität identifiziert die Vorteile und Nachteile, die Menschen aufgrund dieser Kombination von Faktoren erleben.'
```
Comment (robinaly) [inline translations/de.yaml]:
```suggestion
      definition: 'Ein analytisches Framework, um zu verstehen, wie verschiedene Aspekte der sozialen und politischen Identität einer Person (z. B. Geschlecht, Rasse, Klasse, Sexualität, Fähigkeit, Aussehen usw.) zusammentreffen und einzigartige Kombinationen von Diskriminierung und Privileg erzeugen. Intersektionalität identifiziert die Vorteile und Nachteile, die Menschen aufgrund dieser Kombination von Faktoren erleben.'
```

File: translations/de.yaml [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+    latino-a:
+      title: 'Latino/a'
+      definition: 'Eine Person, die aus Lateinamerika stammt oder mit diesem Erbe verbunden ist.'
+    latinx:
+      title: 'Latinx'
+      definition: 'Ein geschlechtsneutrales oder nicht-binäres Alternativativ für Latino oder Latina.'
```
Comment (robinaly) [inline translations/de.yaml]:
```suggestion
      definition: 'Eine geschlechtsneutrale oder nicht-binäre Alternativative für Latino oder Latina.'
```

File: translations/de.yaml [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+    social-justice:
+      title: 'Soziale Gerechtigkeit'
+      definition: 'Eine Vision einer Gesellschaft, in der die Verteilung von Ressourcen gerecht ist und alle Mitglieder physisch und psychologisch sicher und geschützt sind.'
+    transgender:
+      title: 'Transgender'
+      definition: 'Jemand, dessen <a href="#Gender Identity">Geschlechtsidentität</a> von dem abweicht, was typischerweise mit dem bei der Geburt zugewiesenen Geschlecht verbunden ist. Transgender impliziert keine Form von sexueller Orientierung. Oft abgekürzt als trans.'
+    white-privilege:
+      title: 'Weißes Privileg'
+      definition: 'Bezieht sich auf den ungefragten und unverdienten Satz von Vorteilen, Ansprüchen, Leistungen und Entscheidungen, die Menschen allein deshalb gewährt werden, weil sie "weiß" sind, insbesondere wenn sie sich sonst unter denselben sozialen, politischen oder wirtschaftlichen Umständen befinden.'
```
Comment (robinaly) [inline translations/de.yaml]:
```suggestion
      definition: 'Bezieht sich auf den ungefragten und unverdienten Menge von Vorteilen, Ansprüchen, Leistungen und Entscheidungsbefugnissen, die Menschen allein deshalb gewährt werden, weil sie "weiß" sind, insbesondere wenn sie sich sonst unter denselben sozialen, politischen oder wirtschaftlichen Umständen befinden.'
```

**Review by robinaly — CHANGES_REQUESTED** [review]
Only few minor changes. Thanks for pushing this initiative.

File: translations/de.yaml (hunk 1 above)
Comment (MinThaMie) [inline translations/de.yaml]:
```suggestion
  subtitle: 'Bitte überprüfen'
```

**Pull request merged**